### PR TITLE
added missing keywords in step 28 (Create Term Definition)

### PR DESCRIPTION
**Pull request by pchampin**
addresses #261

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-api/pull/267.html" title="Last updated on Dec 20, 2019, 11:38 PM UTC (af16ed0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-api/267/6aa573b...af16ed0.html" title="Last updated on Dec 20, 2019, 11:38 PM UTC (af16ed0)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6901,6 +6907,10 @@ <h4>JsonLdErrorCode</h4>
 <section class="appendix informative" id="changes-from-cr">
   <h2>Changes since Candidate Release of 12 December 2019</h2>
   <ul>
+    <li>Added missing keyworkds <a href="#ctd-invalid-entries">28</a> of the
```
Comment (davidlehn) [inline index.html]:
```suggestion
    <li>Added missing keywords to step <a href="#ctd-invalid-entries">28</a> of the
```

File: index.html [hunk 2]
```diff
@@ -6901,6 +6907,10 @@ <h4>JsonLdErrorCode</h4>
...
+      <a href="#create-term-definition">Create Term Definition algorithm</a>
+      that remaining steps are skipped if the value of `@id` is `null`.
```
Comment (davidlehn) [inline index.html]: This line is a copy & paste from below.  I think it should just be removed and add period above?

**Comment by davidlehn** [comment]
@gkellogg Looks like my comments were not addressed.

File: index.html (hunk 2 above)
Comment (gkellogg) [reply index.html]: Sorry, I'll commit them to master.

Comment (gkellogg) [reply index.html]: Looks like a rebase failure.

**Pull request merged**